docs/docs/ci-templates/gitlab/image-builder.md

```diff
@@ -41,6 +41,10 @@ Make sure to enable the "Expand variable reference" flag.
   - `DOCKER_REGISTRY`: The URL to the Docker registry
   - `DOCKER_REGISTRY_USER`: Username of a techuser with push permission to the Docker registry
   - `DOCKER_REGISTRY_PASSWORD`: Corresponding password of the techuser
+- `T4C_SERVER_REGISTRY`: Registry where you can find the t4c server image
+- `T4C_SERVER_TAG`: Tag that is used for the t4c server image
+- `T4C_SERVER_TEST_DATA_REPO`: Link to the t4c test data repo needed to run the backup tests.
```

Comment:
```suggestion
- `T4C_SERVER_TEST_DATA_REPO`: Link to the Git repository containing t4c test data needed to run the backup tests.
```

As we don't provide the repository, how does it look like?

Reply:
Added a second sentence describing the general structure (i.e., `/data/$CAPELLA_VERSION/repositories/test-repo`)